app/views/pages/search_item.js.erb

```diff
@@ -1,3 +1,12 @@
 $("#search-items-result").empty();
 $("<%= escape_javascript(render 'pages/search_result', items: items) %>")
   .appendTo("#search-items-result");
+
+$('select[name="item[category_id]"]').on('change', function() {
+  const classNames = {};
+  <% Category.all.each do |ctgy| %>
+    classNames[<%= ctgy.id %>] = "<%= CategoriesHelper::css_class_name(ctgy) %>";
+  <% end %>
+  const imageContainer = $(this).parent().parent().prev();
```

Comment:
can this generate an error?

Reply:
Only if the DOM tree is malformed, but the only side-effect woud be not changing the element class.